peer/store-utils.go

```diff
@@ -79,12 +79,12 @@ func GetPeers() ([]Peer, error) {
 	if err != nil || pairs == nil {
 		return nil, err
 	}
-	// If there is only one entry in the store this indicates its a stand
-	// alone cluster, no need to return back it's own information
-	if len(pairs) == 1 {
+	if len(pairs) == 0 {
```

Comment:
I have removed the second call to `InitMyUUID()` in `context/global.go`. I can see the peer details on startup.